models/donation.model.js

```diff
@@ -1,14 +1,15 @@
 const mongooose = require('mongoose');
 
 const DonationSchema = new mongooose.Schema({
-  donor: { type: mongooose.ObjectId, required: true },
-  ngo: { type: mongooose.ObjectId, required: true },
-  isDelivered: { type: Boolean, default: false },
+  user: { type: mongoose.Schema.Types.ObjectId, ref: "User" },
+  ngo: { type: mongoose.Schema.Types.ObjectId, ref: "NGOs" },
+  address: { type: String, require: true },
+  current_state: { type: String, default: "CREATED" },
```

Comment:
Need to add one more field as something like pickup_person storing the name/id of a person responsible for collecting the donation.

Reply:
Done